manage.sh

```diff
@@ -69,6 +69,10 @@ elif [ "$command" == "reset_db" ]; then
     flask db upgrade
 elif [ "$command" == "log" ]; then
     cat logs/server_log
+elif [ "$command" == "validate_story" ]; then
+    python3 validate_story.py
+elif [ "$command" == "visualize_story" ]; then
+    python3 visualize_story_graph.py
```

Comment:
I made a commit on `65-fix-story-graph` for this - doesn't belong in this merge request
also, you should activate the venv first and source the env-vars, but that's all done on said branch - check it out ;)